app/controllers/fitbit_profiles_controller.rb

```diff
@@ -104,28 +104,28 @@ def show
       end
     end
   end
-  
+
   def dump
     @fitbit_profile = FitbitProfile.find_by_id(params[:id]) || not_found
-    Sidekiq::Client.enqueue(FitbitDump,current_user.email,@fitbit_profile.id)
+    Sidekiq::Client.enqueue(FitbitDump,@fitbit_profile.id,current_user.id)
```

Comment:
Instead of `Sidekiq::Client.enqueue(FitbitDump, ...)`, you can use `FitbitDump.perform_async(...)`. `enqueue` is somewhat of a legacy API.